kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -706,7 +706,7 @@ function setAndSaveCurrentExamAttemptLog(store, contentId, itemId, currentAttemp
 }
 
 function closeExam(store) {
```

Comment:
I prefer to leave the state modification in the action for now, and routing the component, but I think it's a matter of taste.

The above one was not working before :)